app/presenters/task_list/status_tag.rb

```diff
@@ -1,32 +1,43 @@
 module TaskList
-  class StatusTag < BaseRenderer
+  class StatusTag
     attr_reader :status
 
-    DEFAULT_CLASSES = %w[govuk-tag app-task-list__tag].freeze
-
     STATUSES = {
       TaskStatus::COMPLETED => nil,
-      TaskStatus::IN_PROGRESS => 'govuk-tag--blue',
-      TaskStatus::NOT_STARTED => 'govuk-tag--grey',
-      TaskStatus::UNREACHABLE => 'govuk-tag--grey',
-      TaskStatus::NOT_APPLICABLE => 'govuk-tag--grey',
+      TaskStatus::IN_PROGRESS => 'light-blue',
+      TaskStatus::NOT_STARTED => 'blue',
+      TaskStatus::UNREACHABLE => nil,
+      TaskStatus::NOT_APPLICABLE => nil
     }.freeze
 
```

Comment:
See for https://design-system.service.gov.uk/patterns/complete-multiple-tasks/ for reasoning for changing the colours here:

> Once the user has completed the task, the status should show as ‘Completed’ and be black text with no background colour. This will draw more attention to tasks that require action.